src/components/wallet/methods/updateAccount.ts

```diff
@@ -12,31 +15,32 @@ export default async function updateAccount() {
       .accountId(this.account.publicKey)
       .call()
       .then((account) => {
-        account.balances = loMap(account.balances, (balance) =>
-          loOmit(balance, [
-            'limit',
-            'buying_liabilities',
-            'selling_liabilities',
-            'is_authorized',
-            'last_modified_ledger',
-            balance.asset_type !== 'native' ? 'asset_type' : null,
-          ])
-        )
+        // account.balances = loMap(account.balances, (balance) =>
+        //   loOmit(balance, [
+        //     'limit',
+        //     'buying_liabilities',
+        //     'selling_liabilities',
+        //     'is_authorized',
+        //     'last_modified_ledger',
+        //     balance.asset_type !== 'native' ? 'asset_type' : null,
+        //   ])
+        // )
 
         this.account = {
           ...this.account,
           state: loOmit(account, [
             'id',
             '_links',
-            'sequence',
-            'subentry_count',
-            'last_modified_ledger',
-            'flags',
-            'thresholds',
+            // 'sequence',
+            // 'subentry_count',
+            // 'last_modified_ledger',
+            // 'flags',
+            // 'thresholds',
             'account_id',
-            'signers',
+            // 'balances',
+            // 'signers',
             'paging_token',
-            'data_attr',
+            // 'data_attr',
           ]),
         }
       })
```

Comment:
Do these changes feel right? Would like to nuke the comments but unsure if this is the right list for the raw output.

Reply:
yeah i think these are a good set of properties to keep.  What is data_attr?  Definitely should omit _links.

Reply:
i'll push

Reply:
`data_attr` is the managed data on an account:
https://www.stellar.org/developers/guides/concepts/list-of-operations.html#manage-data